doc/getting-started/getting-started.ipynb

```diff
@@ -150,146 +150,22 @@
       "==============================\n",
```

Comment:
Please, check this file. I think you have to reset changes here.

Reply:
Done